src/pages/Download/InterPro/index.js

```diff
@@ -0,0 +1,251 @@
+// @flow
+import React from 'react';
+
+import Link from 'components/generic/Link';
+import { foundationPartial } from 'styles/foundation';
+
+import ipro from 'styles/interpro-new.css';
+import ebiGlobalStyles from 'ebi-framework/css/ebi-global.css';
+import fonts from 'EBI-Icon-fonts/fonts.css';
+
+const f = foundationPartial(ebiGlobalStyles, fonts, ipro);
+
+const InterProDownloads = () => {
+  const IS_UNIPARC_READY = true;
+  const UniparcLink = IS_UNIPARC_READY ? Link : 'span';
```

Comment:
If `IS_UNIPARC_READY` is always `true`, maybe we don't need it.

Reply:
Yeah I was trying to remember about that one. I think it was something done to be ready for a change, but I'm assuming that change has already happen and it just stayed there... I'll remove it